src/requests/getForecast.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation
+) => {
+  let endpoint = "https://mcr-codes-weather-app.herokuapp.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  axios.get(endpoint).then((response) => {
```

Comment:
You will need some error handling here, I see you havent gotten to this step though :)